.travis.yml

```diff
@@ -12,4 +12,4 @@ install:
   - pip install -r requirements.txt
 
 addons:
-  postgresql: 9.6
+  postgresql: "9.6"
```

Comment:
[From the looks of it](https://docs.travis-ci.com/user/database-setup/#PostgreSQL) this also requires:

``` yaml
services:
  - postgresql
```